extension/app/ts/AddressBook.tsx

```diff
@@ -135,7 +133,7 @@ export function ListElement(entry: ListElementParam) {
 						<button class = 'card-header-icon' style = 'padding: 0px; margin-left: auto;' aria-label = 'delete'>
 							<span class = 'icon' style = 'color: var(--text-color);' onClick = { entry.type != 'empty' && entry.category === 'My Active Addresses' ? () => entry.removeEntry(entry) : () => {} }> X </span>
 						</button>
-						<button class = 'button is-primary is-small' onClick = { () => {} } disabled = { true }>Edit</button>
+						<button class = 'button is-primary is-small' onClick = { entry.type != 'empty' ? () => entry.renameAddressCallBack(entry) : () => {} }>Edit</button>
```

Comment:
No more disabled attribute being set, I'm not sure how often ``entry.type`` is ``'empty'``  but if its not almost never or not only for a few moments before data is loaded then you may want to have it disabled for help with styling.

Reply:
its always empty when its fetching data from background page, which should take short while and making them disabled would just cause blinking